astropy/units/function/core.py

```diff
@@ -50,7 +50,7 @@ class FunctionUnitBase(object):
     function_unit :  `~astropy.units.Unit` or `string`
         By default, the same as the function unit set by the subclass.
     """
-    # vvvvv the following four need to be set by subclasses
```

Comment:
Hmm, those "vvvvv" were there on purpose; some are matched by "^^^^^" a bit lower down. In principle, I don't mind them being removed if that helps the spell checking (which is very much appreciated!), but do remove the "^^^^" as well!